src/go/rpk/pkg/cli/cmd/acl.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package cmd
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/acl"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/common"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+)
+
+func NewACLCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
+	var (
```

Comment:
should this be a struct shared between many commands since it's inherited among multiple commands if I understand it correctly?

Reply:
It's not shared, but they each have the same basic flags. Since they're all local to each command, I don't know if creating a struct would be better.

Reply:
I was just wondering that if we need to add one more, we would be doing it one place instead of 10